src/chef_authn.erl

```diff
@@ -83,7 +83,13 @@
                           {time, erlang_time() | now} |
                           {path, http_path()} |
                           {get_header, header_fun()}.
-
+-ifdef(otp_18_plus).
```

Comment:
This is correct, it's just a small optimisation, saving a function call when we know it's not necessary. I'm fine with either.